nucypher/blockchain/eth/agents.py

```diff
@@ -820,17 +820,20 @@ def get_fee_amount(self, staker_address: str) -> int:
         fee_amount = self.contract.functions.nodes(staker_address).call()[0]
         return fee_amount
 
-    def get_min_fee_rate_range(self) -> Tuple[int, int, int]:
-        minimum, default, maximum = self.contract.functions.minFeeRateRange().call()
+    def get_fee_rate_range(self) -> Tuple[int, int, int]:
+        """Check maximum, minimum & default fee rate for all stakers and all policies ('global fee range')"""
```

Comment:
Follow the same order of variables that's returned
```suggestion
        """Check minimum, default & maximum fee rate for all stakers and all policies ('global fee range')"""
```